Makefile

```
@@ -147,6 +147,22 @@ golangci-lint: ## Run golangci-lint (requires golangci-lint to be installed)
 		echo "$(YELLOW)golangci-lint not installed. Install with: go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest$(RESET)"; \
 	fi
 
+.PHONY: cyclo
+cyclo: ## Run cyclomatic complexity analysis (requires gocyclo to be installed)
```

checkmake (0.2.2)

[warning] Target body for "cyclo" exceeds allowed length of 5 (13).

(maxbodylength)